packages/core/src/components/Autocomplete.story.tsx

```diff
@@ -62,6 +62,18 @@ storiesOf('Core/Autocomplete', module)
       disabled
     />
   ))
+  .add('Load items on focus.', () => (
+    <Autocomplete
+      accessibilityLabel="Label"
+      name="autocomplete-load-on-focus"
+      label="Label"
+      loadItemsOnFocus
```

Comment:
please move `loadItemsOnFocus` boolean prop to the bottom to be consistent with the rest of the story